app/presenters/nav_tab/global_link_collection.rb

```diff
@@ -0,0 +1,43 @@
+class NavTab::GlobalLinkCollection
+
+  include Rails.application.routes.url_helpers
+
+  attr_reader :ability
+
+  delegate :can?, to: :ability
+
+  def initialize(ability)
+    @ability = ability
+  end
+
+  def links
+    [
+      admin_cross_facility_users,
+      global_settings,
+      admin_cross_facility_billing,
+    ].select(&:present?)
+  end
+
+  private
+
+  def admin_cross_facility_billing
+    if can?(:manage_billing, Facility.cross_facility)
+      global_tab(:admin_billing, facility_transactions_path("all"))
+    end
+  end
+
+  def admin_cross_facility_users
+    if can?(:manage_accounts, Facility.cross_facility) || can?(:manage, User)
```

Comment:
Definitely the `can?(:manage, User)` is wrong here, since it changes depending on facility context, and we don't want that in this case. The ability needs to be: can it manage users in a cross-facility context. Current context doesn't matter. That's why I went with `can?(:manage_accounts, Facility.cross_facility)` and I think I'll have to do something similar for users. But I'm open to ideas.